### PR TITLE
Bump Iglu Server to 0.3.1

### PR DESCRIPTION
**Pull request by oguzhanunlu**

### CODE REVIEW

File: iglu-server/0.3.1/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+FROM snowplow-docker-registry.bintray.io/snowplow/base:0.1.0
```
Comment (BenFradet) [inline iglu-server/0.3.1/Dockerfile]: there is base-debian:0.2.0 now

File: iglu-server/example/docker-compose.yml [hunk 2]
```diff
@@ -13,7 +13,7 @@ services:
         max-size: 1G
   iglu-server:
     container_name: iglu-server
-    image: snowplow-docker-registry.bintray.io/snowplow/iglu-server:0.3.0
+    image: iglu-server:0.3.1-rc2
```
Comment (BenFradet) [inline iglu-server/example/docker-compose.yml]: ?

**Review by BenFradet — COMMENTED** [review]
travis needs to be changed too

File: iglu-server/example/docker-compose.yml (hunk 2 above)
Comment (oguzhanunlu) [reply iglu-server/example/docker-compose.yml]: rc2 image isn't available on our Docker registry but some may need to use a local build until next snowplow-docker release, hence the temporary update here.

**Comment by BenFradet** [comment]
what's the status on this @oguzhanunlu ?

**Comment by oguzhanunlu** [comment]
It is waiting Iglu R10 release. Also, after recent changes in R10, it'll be Iglu Server `0.4.0`. Once r10 is out, I'll update here.

**Comment by BenFradet** [comment]
cool thanks :+1:

**Pull request closed without merging**